conda/recipes/librmm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+# Usage:
+#   conda build -c nvidia -c rapidsai -c conda-forge -c defaults .
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', 'unknown').split('.')[:2]) %}
+package:
+  name: librmm
+  version: {{ version }}
+
+source:
+  # git_url: ../../..
+  path: ../../..
+
+build:
+  number: {{ git_revision_count }}
+  string: cuda{{ cuda_version }}_{{ git_revision_count }}
+  script_env:
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+    - PARALLEL_LEVEL
+    - BUILD_ABI
+
+requirements:
+  build:
+    - cmake >=3.12.4
+
+test:
+  commands:
+    - test -f $PREFIX/include/rmm/rmm.h
```

Comment:
I wonder if `include/rmm/detail/cnmem.h` is necessary here too?

Reply:
Will add it, thanks!